langchain/output_parsers/json.py

```diff
@@ -0,0 +1,15 @@
+import json
+import re
+from typing import Dict
+
+from langchain.schema import OutputParserException
+
+
+def parse_json(text: str) -> Dict:
```

Comment:
can we call this parse_json_markdown (can rename other one)? seems like regex is filtering for that?